sampler/twitter-server.go

```diff
@@ -18,8 +19,13 @@ var (
 	jvmStat   = regexp.MustCompile(`^jvm/uptime$`) // just uptime, for now
 )
 
-func NewTwitterServerSampler(metricsURL string) *TwitterServerSampler {
-	return &TwitterServerSampler{metricsURL}
+func NewTwitterServerSampler(metricsURL, matchRegexp string) *TwitterServerSampler {
+	var matcher *regexp.Regexp = nil
```

Comment:
Tiny nit: nil is already the zero value for pointers, so you could write this as:

```
var matcher *regexp.Regexp
```